os/tools/configure.sh

```diff
@@ -150,6 +150,14 @@ if [ ! -r "${src_config}" ]; then
   exit 6
 fi
 
+if [ -r ${dest_config} ]; then
+  if [ -x "${TOPDIR}/.version" ]; then
```

Comment:
Is this -x? not -r? .version and .config have same usages which includes configured value. .config is checked with -r at above line. But this line checks with -x. It is little strange.

Reply:
typo mistake perhaps